app/views/layouts/_admin_dropdown.html.erb

```diff
@@ -103,7 +103,7 @@
         <ul class="dropdown-menu">
           <% Council.titles.each do |council| %>
             <% if can? :manage, council %>
-              <%= content_tag(:li, link_to(council, edit_council_path(council))) %>
+              <li><%= link_to(council, edit_admin_council_path(council)) %></li>
```

Comment:
I don't like submenues... And you can't access `admin/councils/index` from the menu